### DISCUSSION TITLE
develop/detection/tools/polygon_zone/

### DISCUSSION THREAD

**Discussion by giscus** [roboflow/supervision#1606] [Docs]
# develop/detection/tools/polygon_zone/

A set of easy-to-use utilities that will come in handy in any computer vision project.

https://supervision.roboflow.com/develop/detection/tools/polygon_zone/

**Comment by Muhammad-237** [comment]
there is no code snippet for polygone zone how can we understand it

**Reply by LinasKo** [reply]
[Added](https://supervision.roboflow.com/develop/detection/tools/polygon_zone/). It'll be visible in the next release.

Thanks for reporting it!

**Comment by tonmoy-TS** [comment]
The  PolygonZone attribute 'triggering_position' is now changed to 'triggering_anchors'

**Comment by amammi** [comment]
There's no more frame_resolution_wh parameter in the constructor

**Reply by SkalskiP** [reply]
Hi @amammi! That's right. We've simplified the `PolygonZone` API. You no longer need to pass `frame_resolution_wh` in the constructor.

**Comment by bgorichs** [comment]
My zone.trigger(detections) boolean numpy array is True when a detection is present but is Empty when no detection is present. Shouldn't it be False and not Empty?

**Reply by bgorichs** [reply]
Disregard, it's Empty when there are no detections in frame but False if the detection is in frame but not in the zone.
